scalafmt-cli/src/main/scala/org/scalafmt/cli/ScalafmtRunner.scala

```diff
@@ -55,8 +55,12 @@ trait ScalafmtRunner {
     val files = options.fileFetchMode match {
       case m @ (GitFiles | RecursiveSearch) =>
         val fetchFiles: AbsoluteFile => Seq[AbsoluteFile] =
-          if (m == GitFiles) options.gitOps.lsTree(_)
-          else FileOps.listFiles(_)
+          if (m == GitFiles)
+            options.gitOps
+              .lsTree(_)
+              .filter(file => FileOps.isRegularFile(file.jfile))
```

Comment:
I think it's ok to move this `filter` expression into `GitOpsImpl.lsTree` 👍

Reply:
Done. 🙂

Reply:
Done.